test/ssltest_old.c

```diff
@@ -1915,8 +1915,8 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
             if (cw_num > 0) {
                 /* Write to server. */
 
-                if (cw_num > (long)sizeof cbuf)
-                    i = sizeof cbuf;
+                if (cw_num > (long)sizeof(cbuf))
+                    i = (int)sizeof(cbuf);
```

Comment:
That cast should not be part of this PR.  They are also a subject of discussion, and should not be added spuriously.